p2p/server.go

```diff
@@ -156,6 +156,10 @@ type Config struct {
 	Logger log.Logger `toml:",omitempty"`
 
 	clock mclock.Clock
+
+	// TxArrivalWait is the duration (ms) that the node will wait after seeing
+	// an announced transaction before explicitly requesting it
+	TxArrivalWait int
```

Comment:
the same as before with time.Duration type

Reply:
updated to time.Duration

commit [3c23de2](https://github.com/thogard785/bor/commit/3c23de2486a3eaee9f938aef883ee26c551c1dee)